testsuite/tests/grafana/test_dashboards.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for Grafana Dashboards definitions"""
+import pytest
+
+from testsuite.capabilities import Capability
+
+pytestmark = [
+        pytest.mark.skipif("TESTED_VERSION < Version('2.9')"),
+        pytest.mark.required_capabilities(Capability.OCP4),
+    ]
+
+# pylint: disable=line-too-long
+DASHBOARDS = [
+    ("apicast-mainapp",
+     "sum(rate(nginx_http_connections{namespace='$namespace', pod=~'apicast-$env.*'}[1m])) by (state)"),
+    ("apicast-services",
+     "sum(rate(upstream_status{namespace='$namespace', pod=~'apicast-$env.*', service_id='$service_id'}[1m])) by (status)"),  # noqa: E501
+    ("backend",
+     "sum(rate(apisonator_listener_response_codes{namespace=\\\"$namespace\\\",request_type=\\\"authorize\\\"}[1m]))"),  # noqa: E501
```

Comment:
If you use `'` you can get rid of at least one \ for each escaped quote.

Reply:
rewritten to use `'` in strings that contains `"`